Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2251,9 +2251,8 @@ extension BrowserViewController {
             handleFirstTimeUserActions()
         } else if User.shared.showsRebrandIntro {
             presentRebrandIntro()
-        } else {
-            presentInsightfulSheetsIfNeeded()
         }
+        presentInsightfulSheetsIfNeeded()
```

Comment:
What will be different when moving it outside of the `else`? Wouldn't there be a risk of it trying to present multiple intros?

Reply:
Thought about it twice actually. Will remove it that was part of the improvements on this page.